src/server.js

```diff
@@ -72,7 +74,10 @@ const initServer = async function (db, config) {
     port: config.listenPort,
     routes: {
       validate: {
-        failAction: failActionHandler
+        options: ErrorHandler.validateRoutes(),
+        failAction: async (request, h, err) => {
```

Comment:
I think I just had `failActionHandler` here for the improved test coverage it offered by abstracting it out. No biggie though.

Reply:
ok that's cool i can move it out to that util file